### DISCUSSION TITLE
Enterprise Trial questions

### DISCUSSION THREAD

**Discussion by johntdyer** [emqx/emqx#11552] [General]
hello,
  I was at GCP Next and one of your advocates said that there was a way to trial the enterprise version with a 100 device limit, but I dont recall how exactly to do it.  Could you please help explain the process ?  Thank you and love the product

**Comment by id** [comment]
Hi @johntdyer.

You don't need to do anything special, trial license is included in emqx-enterprise distribution https://www.emqx.com/en/products/emqx.
